test/async-hooks/test-callback-error.js

```diff
@@ -1,63 +1,69 @@
 'use strict';
-
 const common = require('../common');
 const assert = require('assert');
 const spawnSync = require('child_process').spawnSync;
-const async_hooks = require('async_hooks');
+const ah = require('async_hooks');
```

Comment:
+1 to `async_hooks`, the brevity of `ah` doesn't lend itself to maintainability when future people go wandering through this file.